src/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
```suggestion
  background-position: center;
  background-size: 80%;
```
To fix tests

Reply:
i gon't understand why i gotta add background-size, 'cause the stars break down when i do it